docs/source/tutorials/02_build_pipelines.ipynb

```diff
@@ -772,7 +773,7 @@
     {
      "data": {
       "text/plain": [
-       "LineaArtifact(name='iris_model', _version=0)"
+       "LineaArtifact(name='iris_model', _version=27)"
```

Comment:
not sure if such high version numbers become alarming for users. lets start with a clean artifact store for the demos and only include as many versions as there are artifact calls inside the demos.

Reply:
Great point!